app/scripts/modules/core/src/artifact/react/ExpectedArtifactEditor.tsx

```diff
@@ -127,6 +127,9 @@ export class ExpectedArtifactEditor extends React.Component<
     const EditCmp = kind && kind.editCmp;
     return (
       <>
+        <StageConfigField label="Display Name" fieldColumns={fieldColumns} groupClassName={fieldGroupClassName}>
+          <input className="form-control input-sm" value={expectedArtifact.displayName} />
```

Comment:
Is the intention for this field to be read-only?  I wouldn't expect that the user's edits to this input box would have any effect on the saved artifact.  React doesn't implement data binding the way that Angular might here.  I think you'd need to include an `onChange` handler to save modifications to the displayName in `this.state.expectedArtifact`.  Something like:

```jsx
// method on the class
public onDisplayNameEdit = (e: React.FormEvent<HTMLInputElement>) => {
  const newName = e.value;
  const expectedArtifact = { ...this.state.expectedArtifact };
  expectedArtifact.displayName = newName;
  this.setState({ expectedArtifact });
};

// in the render() method...
  <input className="..." value={expectedArtifact.displayName} onChange={this.onDisplayNameEdit} />
```

Reply:
Good point. Added with 5d313f424.